src/components/dialogs/network-modifications/line-attach-to-voltage-level/line-attach-to-voltage-level-dialog.js

```diff
@@ -249,6 +256,18 @@ const LineAttachToVoltageLevelDialog = ({
         [setValue]
     );
 
+    useEffect(() => {
+        if (studyUuid && currentNode?.id) {
+            fetchVoltageLevelsListInfos(studyUuid, currentNode?.id).then(
+                (values) => {
+                    setVoltageLevelOptions(
+                        values.sort((a, b) => a?.id?.localeCompare(b?.id))
+                    );
+                }
+            );
+        }
+    }, [studyUuid, currentNode?.id]);
+
```

Comment:
same code bloc in `components/dialogs/network-modifications/line-split-with-voltage-level/line-split-with-voltage-level-dialog.js` 
please find a way to create only one function.